integration-tests/chopsticks/src/network/spiritnet.ts

```diff
@@ -0,0 +1,60 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import * as HydraDxConfig from './hydroDx.js'
+
+export const options: SetupOption = {
+	endpoint: 'wss://kilt-rpc.dwellir.com',
+	db: './db/spiritnet.db.sqlite',
+	port: 8002,
+	wasmOverride: '../../target/debug/wbuild/spiritnet-runtime/spiritnet_runtime.wasm',
+	allowUnresolvedImports: true,
+}
+
+export const defaultStorage = (addr: string) => ({
+	technicalCommittee: { Members: [addr] },
+	council: { Members: [addr] },
+	System: {
+		Account: [[[addr], { providers: 1, data: { free: 1000 * 1e12 } }]],
+	},
+	polkadotXcm: {
+		safeXcmVersion: 3,
+	},
+})
+
+export const paraId = 2086
+export const hydraDxDestinationV2 = {
+	V2: {
```

Comment:
We can't use `V3`?

Reply:
I added another test case for v3: https://github.com/KILTprotocol/kilt-node/pull/614/files#diff-ce4305e382897732899a9335dad6b6d4350a23b2d2be29fc5c4ca6f864806a48R11-R62